lib/SPIRV/VectorComputeUtil.cpp

```diff
@@ -0,0 +1,116 @@
+//=- VectorComputeUtil.cpp - vector compute utilities implemetation * C++ -*-=//
+//
+//                     The LLVM/SPIR-V Translator
```

Comment:
Please add the license header as in https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/master/lib/SPIRV/LLVMToSPIRVDbgTran.cpp

Reply:
Fixed